Makefile

```diff
@@ -168,7 +156,7 @@ bootstrap-flow:
 test-flow:
 	$(NODE) scripts/parser-tests/flow
 
-test-flow-ci: build-bundle-ci bootstrap-flow
+test-flow-ci: build-bundle bootstrap-flow
```

Comment:
The `-ci` variant of make rules specifies the correct dependencies of recipes, so `make build-bundle-ci` ensures that it will run after `bootstrap` while `make build-bunle` (run by human) always assume `make bootstrap` or `make watch` has been run. The correct dependencies are critical to `make -j` on CI.

Reply:
Now that `make bootstrap-only` would be the same as `yarn install` on CI, and thus every command needs it, it would always be executed first because it's needed by every job.

For this reason, I simplified the `Makefile` by moving `yarn install` to the Travis and CircleCI config files. This also makes it possible to show then in different "steps" on CircleCI, making it clearer how long they take.

Do you think that it would still be better to move `yarn` back to the `makefile`, and recreate the `-ci` rules I removed?

Reply:
`bootstrap-flow` can run independently with `bootstrap`. Both of them are network operations so it should still be faster than running sequentially.

However, since `bootstrap-flow` is not that slow, I am good if the execution time benefit is marginal.